contracts/ExampleToken.cdc

```diff
@@ -193,38 +219,35 @@ pub contract ExampleToken: FungibleToken {
 
     init() {
         self.totalSupply = 1000.0
-
         self.VaultStoragePath = /storage/exampleTokenVault
         self.ReceiverPublicPath = /public/exampleTokenReceiver
-        self.BalancePublicPath = /public/exampleTokenBalance
+        self.MetadataPublicPath = /public/exampleTokenMetadata
         self.AdminStoragePath = /storage/exampleTokenAdmin
 
-        // Create the Vault with the total supply of tokens and save it in storage
-        //
+        // Create the Vault with the total supply of tokens and save it in storage.
         let vault <- create Vault(balance: self.totalSupply)
         self.account.save(<-vault, to: self.VaultStoragePath)
 
-        // Create a public capability to the stored Vault that only exposes
-        // the `deposit` method through the `Receiver` interface
-        //
+        // Create a public capability to the stored Vault that exposes
+        // the `deposit` method through the `Receiver` interface.
         self.account.link<&{FungibleToken.Receiver}>(
             self.ReceiverPublicPath,
             target: self.VaultStoragePath
         )
 
         // Create a public capability to the stored Vault that only exposes
-        // the `balance` field through the `Balance` interface
-        //
-        self.account.link<&ExampleToken.Vault{FungibleToken.Balance}>(
-            self.BalancePublicPath,
+        // the `balance` field through the `Balance` interface and also
+        // the `resolveView` method through the MetadataViews `Resolver` interface.
+        self.account.link<&ExampleToken.Vault{FungibleToken.Balance, MetadataViews.Resolver}>(
+            self.MetadataPublicPath,
             target: self.VaultStoragePath
         )
 
         let admin <- create Administrator()
         self.account.save(<-admin, to: self.AdminStoragePath)
 
         // Emit an event that shows that the contract was initialized
-        //
         emit TokensInitialized(initialSupply: self.totalSupply)
     }
 }
+ 
```

Comment:
Extra line break here

Reply:
I learnt to code with C, I like my newlines at EOF jajaja. Deleted!